packages/application-shell/package.json

```diff
@@ -25,6 +25,7 @@
     "npm": ">=5"
   },
   "dependencies": {
+    "@babel/runtime": "7.1.2",
```

Comment:
this is used [here](https://github.com/commercetools/merchant-center-application-kit/blob/7719428fefffa2fbf24128aec7ae8212e96d06f2/packages/application-shell/src/components/quick-access/quick-access.js#L119)

Reply:
What happens without it?

Reply:
you get a warning with the rollup build saying unresolved dep

Reply:
the compiled code looks like

```
...
      var _subCommands = _asyncToGenerator(
      /*#__PURE__*/
      _regeneratorRuntime.mark(function _callee() {

```

Reply:
Hmm ok 🤷‍♂️